book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -8,24 +8,324 @@ This can involve hosting different events that are attended by people with lived
+
+At the event, it is important to have clear communication and identification of staff and organizers especially if their roles are to support attendees. 
+
+You may want to consider having a pdf, website page, or paper copy introducing the event organizers and facilitators so that is it clear who to ask for help. 
+You may want to highlight people who are able to help if an attendee has a question about accommodation, or the location of accessible restrooms as these people may be different from session facilitators and technology support staff. 
+By including contact details and explaining the roles of organizers, it can help attendees feel understood and welcomed instead of being redirected to multiple people who are unable to help them. 
+
+If possible, it can also be helpful to have a dedicated person who can be easily located and contacted during the event that is able to take action if there is a problem. 
+For example, if an elevator is broken, the restroom blocked, accessible doors not opening, or the audio induction loop not working. 
+This person could be available during event registration or at the start of the event to introduce themselves and provide assistance to those who need it. 
+This may not be feasible for all events and may be more important for in-person events, but it is important to consider how to make the event accessible and inclusive throughout. 
+Inclusion can be planned for, but it also needs implementation and support to make it happen.
+
+
+(cl-inclusive-events-planning-post-event)=
+## After the Event
+Consider if you are able to make content from the event accessible to attendees, or open to a wider audience, after the event. 
+
+For events that have recorded sessions, this may be uploading the content to YouTube and ensuring that the captions have been reviewed and added to any video content. 
+Having a human reviewer of captions can be helpful in catching any errors made by automatic captioning software and ensuring that content is of a good quality (such as editing out any audio glitches and flagging triggers for photosensitive epilepsy). 
+
+If your event included content such as slides or documented notes, consider where and how you will archive them so that they can be accessed and referenced by attendees. 
+Online repositories such as Zenodo are commonly used by research communities but are not universally design-compliant and require both technical expertise and knowledge of how to use and access it as well as requiring an internet connection to access content. 
+Consider using Zenodo in addition to an accessible format. 
+This could be a website commonly visited by your attendees, sending post-event summary information to attendees, or providing recordings of sessions that display session content.
+
+Finally, for in-person events and related to safeguarding, consider how you will ensure that everyone is able to access return transport safely. 
```

Comment:
Finally, for in-person events and related to safeguarding, anticipate, and prepare to guarantee everyone can access return transport.
```suggestion
Finally, for in-person events and related to safeguarding, anticipate, and prepare to guarantee everyone can access return transport.
```